package.json

```diff
@@ -258,7 +258,7 @@
     "@grafana/prometheus": "workspace:*",
     "@grafana/runtime": "workspace:*",
     "@grafana/saga-icons": "workspace:*",
-    "@grafana/scenes": "4.27.0",
+    "@grafana/scenes": "^5.3.0",
```

Comment:
This is going to break everything, it will need to backport the new URLSync strategy too

Reply:
See https://github.com/grafana/grafana/pull/89851